bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1210,7 +1210,7 @@ spec:
                   value: /opt/manifests
                 - name: ODH_PLATFORM_TYPE
                   value: OpenDataHub
-                image: REPLACE_IMAGE:latest
+                image: quay.io/opendatahub/opendatahub-operator:latest
```

Comment:
should revert

Reply:
I guess one of the make commands updates this? I made a similar review comment on Luca's PR, but missed it myself here...will revert, thanks!